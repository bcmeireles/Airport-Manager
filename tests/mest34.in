a LIS Portugal Lisboa
a POR Portugal Lisboa
v TT6 LIS POR 14-02-2022 14:00 2:00 1000
v TT7 LIS POR 14-02-2022 14:00 2:00 1000
v TT8 LIS POR 14-02-2022 14:00 2:00 1000
v TT9 LIS POR 14-02-2022 14:00 2:00 1000

r TT6 14-02-2022 123456789A 200
r TT6 14-02-2022 123456789B 200
r TT6 14-02-2022 123456789C 200
r TT6 14-02-2022 123456789D 200
r TT6 14-02-2022 123456789E 200

e 123456789E
e 123456789A
r TT6 14-02-2022
e TT6



q