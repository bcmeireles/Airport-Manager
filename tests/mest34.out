airport LIS
airport POR
123456789B 200
123456789C 200
123456789D 200
